### PR TITLE
sending email for notify reward amount

### PR DESCRIPTION
**Pull request by ae2079**
Add fields for the notify reward amount notification from giv-token-contracts service
related to: https://github.com/Giveth/giveth-dapps-v2/issues/4194

### CODE REVIEW

File: src/services/notificationService.ts [hunk 1]
```diff
@@ -163,6 +163,19 @@ const activityCreator = (payload: any, orttoEventName: NOTIFICATIONS_EVENT_NAMES
...
+        "str:cm:farm": payload.farm,
+        "str:cm:message": payload.message,
+        "str:cm:network": payload.network,
+        "str:cm:script": payload.script,
+        "str:cm:transactionhash": payload.transactionHash,
+      }
+      break
     default:
       logger.debug('activityCreator() invalid event name', orttoEventName)
       return;
```
Comment (ae2079) [inline src/services/notificationService.ts:181]: @mohammadranjbarz 
I think we need to add notification type for the giv-token-contracts service to the database. (based on line 223 of this file that we call getNotificationTypeByEventNameAndMicroservice() function)

Comment (mohammadranjbarz) [reply src/services/notificationService.ts:181]: @ae2079  Yes please do it ( write a migration for that)

File: migrations/1718888344202-seedNotificationTypeForNotifyRewardAmount.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+import { NOTIFICATION_CATEGORY, NOTIFICATION_TYPE_NAMES } from '../src/types/general';
+import { MICRO_SERVICES } from '../src/utils/utils';
+import { NotificationType, SCHEMA_VALIDATORS_NAMES } from '../src/entities/notificationType';
+
+const NotifyRewardAmountNotificationType = [
+    {
+        name: NOTIFICATION_TYPE_NAMES.NOTIFY_REWARD_AMOUNT,
```
Comment (ae2079) [inline migrations/1718888344202-seedNotificationTypeForNotifyRewardAmount.ts:8]: @mohammadranjbarz 
I ran the migration and these fields are set successfully, but I am not sure about the correctness of their values and I don't now that I should set other fields too, or not

**Review by mohammadranjbarz — APPROVED** [review]
Thanks @ae2079, LGTM (we can merge it once we can test it together after deployment)

**Review by RamRamez — CHANGES_REQUESTED** [review]
@ae2079 You need to add `notifyRewardAmount` to `SEGMENT_METADATA_SCHEMA_VALIDATOR` to make it work.

**Comment by ae2079** [comment]
> @ae2079 You need to add `notifyRewardAmount` to `SEGMENT_METADATA_SCHEMA_VALIDATOR` to make it work.

Done, thank you Ramin

**Review by RamRamez — APPROVED** [review]
Thanks @ae2079

**Pull request merged**